doc/faq.rst

```diff
@@ -0,0 +1,229 @@
+-----------------------------
+
+Auto-sklearn can be restricted to only use interpretable models and preprocessing algorithms.
+Please see the section :ref:`Restricting the searchspace` to learn how to restrict the models
+which are searched over or see
+`this example <examples/40_advanced/example_interpretable_models.html>`_.
+
+We don't provide a judgement which of the models are interpretable as this is very much up to the
+specific use case, but would like to note that decision trees and linear models usually most
+interpretable.
+
+Limiting the number of model evaluations
+----------------------------------------
+
+In certain cases, for example for debugging, it can be helpful to limit the number of
+model evaluations. We do not provide this as an argument in the API as we believe that it
+should NOT be used in practice, but that the user should rather provide time limits.
+An example on how to add the number of models to try as an additional stopping condition
+can be found `in this github issue <https://github.com/automl/auto-sklearn/issues/451#issuecomment-376445607>`_.
+Please note that Auto-sklearn will stop when either the time limit or the number of
+models termination condition is reached.
+
+Ensemble contains only a dummy model
+------------------------------------
+
+TODO
+
+3. How to set useful budgets: https://github.com/automl/auto-sklearn/issues/57
+
+Parallel processing and oversubscription
+----------------------------------------
+
+TODO
+
+Meta-Learning
+=============
+
+Which datasets are used for meta-learning?
+------------------------------------------
+
+We updated the list of datasets used for meta-learning several times and this list now differs
+significantly from the original 140 datasets we used in 2015 when the paper and the package were
+released. An up-to-date list of `OpenML task IDs <https://docs.openml.org/#tasks>`_ can be found
+on `github <https://github.com/automl/auto-sklearn/blob/master/scripts/update_metadata_util.py>`_
+
+How can datasets from the meta-data be excluded?
+------------------------------------------------
+
+For *Auto-sklearn 1.0* one can pass the dataset name via the ``fit()`` function. If a dataset
+with the same name is within the meta-data, that datasets will not be used.
+
+For *Auto-sklearn 2.0* it is not possible to do so because of the method used to construct the
+meta-data.
+
+Which meta-features are used for meta-learning?
+-----------------------------------------------
+
+We do not have a user guide on meta-features but they are all pretty simple and can be found
+`in the source code <https://github.com/automl/auto-sklearn/blob/master/autosklearn/metalearning/metafeatures/metafeatures.py>`_.
+
+How is the meta-data generated?
+-------------------------------
+
+We currently generate meta-data the following way. First, for each of the datasets mentioned
+above, we run Auto-sklearn without meta-learning for a total of two days on multiple metrics (for
```

Comment:
See also our paper <link>

Reply:
We don't give these details in the Auto-sklearn 1.0 paper.